sword3client/client.py

```diff
@@ -175,7 +174,6 @@ def _metadata_deposit_properties(
 
         headers = {
             "Content-Type": "application/json; charset=UTF-8",
-            "Content-Length": content_length,
```

Comment:
Should leave this in as we don't want to make any assumption about HTTP client libraries filling in the Content-Length header (even though urllib and requests both do).